### ISSUE TITLE
Cannot execute copilot job override: read dir "overrides/cdk": open templates\overrides\cdk: file does not exist

### ISSUE THREAD

**Issue by RaffaeleSgarro** [aws/copilot-cli#4812]
I cannot execute the command `copilot job override` with the latest v1.27.0. These are the steps to reproduce:

```sh
 copilot job override --name test-override --env prod --tool cdk
```

Expected: to have the CDK application under my job directory in `copilot/`

Actual (error): scaffold CDK application under "C:\\xxx\\yyy\\copilot\\test-override\\overrides": read dir "overrides/cdk": open templates\overrides\cdk: file does not exist

Additional information: neither CDK nor YAML-patches overrides work. The error is the same with `--tool=yamlpatch`

OS: Windows 10

**Comment by Lou1415926** [comment]
Heyy @RaffaeleSgarro ! This looks like the same issue as https://github.com/aws/copilot-cli/issues/4791#issuecomment-1522162900. I've confirmed that this was a bug that happens on Windows. The fix will be rolled out in the next release! Sorry for the trouble : (

**Comment by dannyrandall** [comment]
Hey @RaffaeleSgarro! The fix is now released in [v1.28.0](https://github.com/aws/copilot-cli/releases/tag/v1.28.0) 🎊! Let us know if you have any other issues!

**Comment by RaffaeleSgarro** [comment]
It works like a charm! Thank you ❤️
